webpack/components/NewTemplateSync/components/NewTemplateSyncForm/NewTemplateSyncForm.js

```diff
@@ -74,9 +74,9 @@ const NewTemplateSyncForm = ({
           )(values[syncType]),
           message: `Templates were ${syncType}ed.`,
           item: 'TemplateSync',
-        }).then(args => {
-          history.replace({ pathname: '/template_syncs/result' });
-        });
+          actions,
+          successCallback: () => history.replace({ pathname: '/template_syncs/result' })
+        })
```

Comment:
nit: can you please extract the onSubmit callback outside ?
makes it more readable, plus it won't be re-render every rendering.
```js
const onFormSubmit = (values, actions) => {
    const url = syncType === 'import' ? importUrl : exportUrl;
    return submitForm({
      url,
      values: compose(
         addLocParams,
        addOrgParams
      )(values[syncType]),
      message: `Templates were ${syncType}ed.`,
      item: 'TemplateSync',
      actions,
      successCallback: () => history.replace({ pathname: '/template_syncs/result' })
    })
  return (
    <ForemanForm onSubmit={onFormSubmit} /> 
   )
```